plugins/LilyPad/LilyPad.cpp

```diff
@@ -163,8 +163,8 @@ struct Stick {
 
 // Sum of states of all controls for a pad (Not including toggles).
 struct ButtonSum {
-	int buttons[12];
-	Stick sticks[3];
+	int buttons[16];
+	Stick sticks[2];
 };
 
 #define PAD_SAVE_STATE_VERSION	3
```

Comment:
`static const u32 g_SaveVersion = (0x9A0B << 16) | 0x0000;`
to
`static const u32 g_SaveVersion = (0x9A0C << 16) | 0x0000;`
?